README.md

```diff
@@ -569,6 +569,7 @@ See the [hacking quick-start](https://posita.github.io/numerary/0.3/contrib/#hac
 ## Customers [![``numerary``-encumbered](https://raw.githubusercontent.com/posita/numerary/v0.3.0/docs/numerary-encumbered.svg)](https://posita.github.io/numerary/)
 
 * [``dyce``](https://github.com/posita/dyce/) - a pure-Python library for modeling arbitrarily complex dice mechanics and ~~mother~~ *birthing code base* of ``numerary``!
+* [``phantom-types``](https://github.com/antonagestam/phantom-types) - predicates and other type constraints without runtime overhead
```

Comment:
@antonagestam, I threw together this as a placeholder. Let me know what language you would prefer, or whether you'd prefer not to be added here at all. (No judgment either way. I'll leave it entirely up to you.)

Reply:
Now that's a cute badge! Thanks, I very much appreciate the extra attention for the library. I'll leave it up to you to decide about the formatting though, go with whichever you feel fits best 🙂